src/mscorlib/src/System/PinnableBufferCacheEventSource.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+namespace System
+{
+    [EventSource(Name = "Microsoft-DotNETRuntime-PinnableBufferCache-System")]
+    internal sealed class PinnableBufferCacheEventSource : EventSource
```

Comment:
If we expect users to use these events to diagnose problems, should we also offer a way to configure the various bucket sizes and behaviors in response?

Also because with limited runway available, we might not get the numbers right.

Reply:
Yes, I'm currently exploring that.